src/index.ts

```diff
@@ -4,6 +4,7 @@ import { Adapter } from "./adapter";
 
 export { Adapter } from "./adapter";
 export { DefaultAdapter } from "./adapters/default";
+export { PythAdapter } from "./adapters/pyth";
```

Comment:
maybe here this file is only included if the client explicitly includes, ex. on client side

```
import { PythAdapter } from 'erc7412/adapters/pyth';
```